src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1309,7 +1309,7 @@ FixedwingPositionControl::control_takeoff(const Vector2f &curr_pos, const Vector
 					   _parameters.throttle_max, // XXX should we also set runway_takeoff_throttle here?
 					   _parameters.throttle_cruise,
 					   _runway_takeoff.climbout(),
-					   radians(_runway_takeoff.getMinPitch(pos_sp_curr.pitch_min, 10.0f, _parameters.pitch_limit_min)),
```

Comment:
comment only: additional benefit is removal of hidden hard-coded values like this 10 degree thing...